modules.d/01systemd-networkd/networkd-config.sh

```diff
@@ -18,6 +18,9 @@ for f in /run/systemd/network/*.network; do
         echo "[DHCPv6]"
         echo "RequestOptions=59 60"
     } >> "$f"
+
+    # Remove the default network if at least one was generated
+    rm -f /etc/systemd/network/99-dracut-default.network
```

Comment:
Seems inconsistent to use `"$systemdnetworkconfdir"` in one place and `/etc/systemd/network` in another ? Would it be possible to stay consistent ?

Reply:
Can I just use that there? I somehow thought it'd only be available at "build time".

Reply:
Found where to add it so it becomes available.

Reply:
thanks